packages/http/src/testing/createSimpleServer.helper.ts

```diff
@@ -61,4 +62,3 @@ export const createSimpleServer = ({
     close: () => new Promise((resolve, reject) => server.close((err: any) => err ? reject(err) : resolve())),
   };
 };
-
```

Comment:
Where's newline?

Reply:
![obraz](https://user-images.githubusercontent.com/23040671/73426089-c2dedd00-4333-11ea-8346-5459e1fbeaaa.png)
🤔

Reply:
Ok, so that was a duplicate trailing newline. 👍